plugins/main/tap/tap.go

```diff
@@ -210,6 +211,22 @@ func createTap(conf *NetConf, ifName string, netns ns.NetNS) (*current.Interface
 		}
 		tap.Name = ifName
 
+		if conf.Bridge != "" {
```

Comment:
why don't you add this block after re-fetching the link and set it up? You'll save one LinkByName.

Reply:
Good catch, I didn't notice it. 

Thanks for pointing it out.
@mlguerrero12